src/raftstore/store/store.rs

```diff
@@ -110,14 +114,16 @@ pub fn create_event_loop<T, C>(cfg: &Config) -> Result<EventLoop<Store<T, C>>>
 }
 
 impl<T: Transport, C: PdClient> Store<T, C> {
-    pub fn new(sender: Sender<Msg>,
+    pub fn new(chan: (Sender<Msg>, Receiver<SnapshotStatusMsg>),
```

Comment:
why not using a structure to contain all args to avoid `too many args` ? 
I don't think `chan` is a good style here.